_posts/12-05-01-Building-your-Application.md

```diff
@@ -23,34 +23,34 @@ Među zadacima koje biste možda želeli da automatizujete su:
 
 ### Alati za automatizaciju izgradnje
 
-Alati za izgradnju mogu biti opisani kao skup skripti koje rukuju zajedničkim zadacima razvoja softvera. Alat za
-zgradnju nije deo vašeg softvera, on deluje na vaš softver 'izvana'.
+Alati za izgradnju (build) mogu biti shvaćeni kao skup skripti koje se bave određnim učestalim zadacima isporuke (deploy-a) softvera. 
+Alat za izgradnju nije deo vašeg softvera, on deluje na vaš softver 'izvana'.
 
-Postoji mnogo dostupnih alata otvorenog koda da vam pomogne sa automatizacijom, neki su napisani u PHP-u, drugi nisu.
-To ne treba da vas sputava da ih koristite, ako više odgovaraju za određeni posao. Evo nekoliko primera:
+Postoji puno alata otvorenog koda koji vam mogu pomoći pri automatizovanoj izgradnji (build), neki su napisani u PHP-u ali 
+postoje oni koji nisu. To ne treba da vas sputava da ih koristite, ako više odgovaraju za određeni posao. Evo nekoliko primera:
 
-[Phing](http://www.phing.info/) je najlakši način da počnete sa primenom automatizacije isporuivanja koda u svetu PHP-a. Pomoću Phing-a
-možete da kontrolišete pakovanje, angažovanje(deployment) ili testiranje procesa unutar jednostavne XML build datoteke. Phing (koji se zasniva na [Apache Ant](http://ant.apache.org/)) obezbeđuje bogat set zadataka, obično potrebnih za instalaciju ili ažuriranje web aplikacija i može se proširiti dodatnim prilagođenim zadacima, pisanim u PHP-u.
+[Phing](http://www.phing.info/) je najlakši način da počnete sa primenom automatizacije isporučivanja koda u svetu PHP-a. Pomoću Phing-a
+možete da kontrolišete pakovanje, isporuku ili testiranje procesa unutar jednostavne XML build datoteke. Phing (koji se zasniva na [Apache Ant](http://ant.apache.org/)) obezbeđuje bogat set zadataka, obično potrebnih za instalaciju ili ažuriranje web aplikacija i može se proširiti dodatnim prilagođenim zadacima, pisanim u PHP-u.
 
 [Capistrano](https://github.com/capistrano/capistrano/wiki) je sistem namenjen *srednjim-do-naprednim programerima* za
-izvršavanje komandi na struktuiran, ponovljivi način na jednoj ili više udaljenih mašina. Prekonfigurisan je za primenu (deploying) na Ruby on Rails applikacijama, međutim ljudi **uspešno angažuju(deploying) PHP sisteme** sa njim. Uspešno korišćenje Capistrana zavisi od radnog iskustva sa Ruby i Rake.
+izvršavanje komandi na struktuiran, ponovljivi način na jednoj ili više udaljenih mašina. Prekonfigurisan je za primenu na Ruby on Rails applikacijama, međutim ljudi **uspešno isporučuju PHP sisteme** sa njim. Uspešno korišćenje Capistrana zavisi od radnog iskustva sa Ruby i Rake.

```

Comment:
ovde mi je nešto sumnjivo, tačnije to "struktuiran, ponovljivi način"... Ja bih čak izbacio to "ponovljiv" ako je zbunjujuće, tj ako nije korisno kao informacija.